evm/src/main/java/org/hyperledger/besu/evm/precompile/P256VerifyPrecompiledContract.java

```diff
@@ -123,7 +123,7 @@ public PrecompileContractResult computePrecompile(
       final Bytes input, final MessageFrame messageFrame) {
     if (input.size() != 160) {
       LOG.warn(
-          "Invalid input length for P256VERIFY precompile: expected 128 bytes but got {}",
+          "Invalid input length for P256VERIFY precompile: expected 160 bytes but got {}",
```

Comment:
Good catch, but what if you create a constant for the value, even if the value is not meant to change it would have avoid the error in the first place, so it is a good practice.